langchain4j-mongodb-atlas/src/test/java/dev/langchain4j/store/embedding/mongodb/MongoDbEmbeddingStoreIT.java

```diff
@@ -0,0 +1,57 @@
+package dev.langchain4j.store.embedding.mongodb;
+
+import com.mongodb.client.MongoClient;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreWithFilteringIT;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.mongodb.MongoDBAtlasLocalContainer;
+
+import static dev.langchain4j.store.embedding.mongodb.TestHelper.*;
+
+public class MongoDbEmbeddingStoreIT extends EmbeddingStoreWithFilteringIT {
+
+    public static class ContainerIT extends MongoDbEmbeddingStoreWithRemovalIT {
+        static MongoDBAtlasLocalContainer mongodb = new MongoDBAtlasLocalContainer("mongodb/mongodb-atlas-local:7.0.9");
+
+        @BeforeAll
+        static void start() {
+            TestHelper.assertDoContainerTests();
+            mongodb.start();
+        }
+
+        @AfterAll
+        static void stop() {
+            mongodb.stop();
+        }
+
+        @Override
+        MongoClient createClient() {
+            return createClientFromContainer(mongodb);
+        }
+    }
+
+    private TestHelper helper = new TestHelper(createClient()).initialize();
```

Comment:
I think we should make all `TestHelper` static, in order to reuse the database and collection.

Reply:
Replied in the [above comment](https://github.com/langchain4j/langchain4j/pull/1869#discussion_r1796161921).